test/util/range.cpp

```diff
@@ -213,6 +213,21 @@ void test_pointers()
     BOOST_CHECK(bgr::at(r2, 5) == 5);
 }
 
+void test_const_pointers()
```

Comment:
The reason I'm suggesting these changes is that, with current system, I overlooked the failing test (the missing const of PR#243) last time...